pylint/checkers/base/docstring_checker.py

```diff
@@ -191,12 +191,13 @@ def _check_docstring(
                     # Strings.
                     if func.bound.name in {"str", "unicode", "bytes"}:
                         return
-            if node_type == "module":
-                message = "missing-module-docstring"
-            elif node_type == "class":
-                message = "missing-class-docstring"
-            else:
-                message = "missing-function-docstring"
+            match node_type:
+                case "module":
+                    message = "missing-module-docstring"
+                case "class":
+                    message = "missing-class-docstring"
+                case _:
+                    message = "missing-function-docstring"
```

Comment:
(It's old code from me, in fact one of my first PR in pylint), I wonder why I didn't use the node directly and introduced a variable for the node type. Might fix that in another refactor.

Reply:
> I wonder why I didn't use the node directly and introduced a variable for the node type. Might fix that in another refactor.

Can't exactly read your mind. However, doing constant isinstance checks might not be the best solution. Adding a temporary variable with `node_type` seems to do the trick.